drivers/gasket/vars.yaml

```diff
@@ -0,0 +1 @@
+VERSION: "{{ .BUILD_ARG_TAG }}"
```

Comment:
I have mixed feelings about what to do for the version. I see in other places we have the library/module/app/etc version as a var and are using that (sometimes with this build arg tag). The issues I have with that are:

1. The version we put in the vars may or may not actually match the version of the module in the pkg. It seems like we would be better off extracting the version from somewhere in the pkg?
2. The gasket-driver doesn't actually have versions. It rarely changes, but the best we have is a commit sha.

Any thoughts or preferences here?

Reply:
> The version we put in the vars may or may not actually match the version of the module in the pkg. It seems like we would > be better off extracting the version from somewhere in the pkg?

it depends, sometimes it might be completely impossible to infer that from `pkgs` so care has to be taken care while updating pkgs version. https://github.com/siderolabs/extensions/blob/main/Pkgfile#L8 we do the same here.

> The gasket-driver doesn't actually have versions. It rarely changes, but the best we have is a commit sha.

we could use the short commit sha

Reply:
I'll see if it's not too much work to extract this data out of pkgs.